vllm_ascend/attention/attention_v1.py

```diff
@@ -141,24 +141,22 @@ def reorder_batch(self, input_batch: "InputBatch",
 
     def build(self, num_reqs, num_actual_tokens, max_query_len,
               common_prefix_len):
-        if vllm_version_is("0.8.5") or vllm_version_is("0.8.5.post1"):
```

Comment:
need rebase, this change is merge to main already. https://github.com/vllm-project/vllm-ascend/commit/7aa4f85f10d55ca1b4a97f86938c9ef9e9706e44